grafana/instance/instance.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: rhel-edge
+  namespace: grafana-operator
+spec:
+  # TODO: this hard-coded image version is necessary until the currently available version
+  # of grafana operator from OperatorHub (v4.7.1) pulls in a later version of grafana
+  baseImage: 'docker.io/grafana/grafana:8.5.5'
+  ingress:
+    enabled: true
+  config:
+    auth:
+      disable_signout_menu: true
+    auth.anonymous:
+      enabled: true
+    log:
+      level: warn
+      mode: console
+    security:
+      admin_password: "rhel"
+      admin_user: "rhel"
```

Comment:
change the password/user  to `sigstore-rh/sigstore-rh` or something like that, with a note to the user to update them to something more secure. Also, I'll check on the comment above (#L7,8) for what version of grafana  is deployed by default with current grafana-operator.